src/NuGet.Core/NuGet.ProjectModel/LockFile/LockFileFormat.cs

```diff
@@ -370,7 +370,7 @@ private static JObject WriteLogMessage(IAssetsLogMessage logMessage, string proj
         /// </summary>
         /// <param name="json"><code>JObject</code> containg the json representation of the log message.</param>
         /// <returns><code>IAssetsLogMessage</code> representing the log message.</returns>
-        private static IAssetsLogMessage ReadLogMessage(JObject json, string projectPath)
+        internal static IAssetsLogMessage ReadLogMessage(JObject json, string projectPath)
```

Comment:
This is now called in CacheFileFormat to write the log messages to the cache file.  I was told to leave the log messages in the assets file in case anyone was depending on them.  And rather than copy/paste this code, I just made this method internal so that CacheFileFormat could call it.